drivers/I2C.h

```diff
@@ -188,6 +198,10 @@ class I2C : private NonCopyable<I2C> {
     static I2C  *_owner;
     int         _hz;
     static SingletonPtr<PlatformMutex> _mutex;
+    gpio_t _gpio_sda;
```

Comment:
should these be local to the reset function? Not sure why else to make the class bigger.